packages/compass-query-bar/src/components/query-option.tsx

```diff
@@ -79,6 +81,9 @@ export const documentEditorLabelContainerStyles = css(
   }
 );
 
+// Data Explorer limits for maxTimeMS (5 minutes = 300,000ms)
+const WEB_MAX_TIME_MS_LIMIT = 300_000; // 5 minutes
```

Comment:
I'd suggest storing this in a single place + explaining _why_ 5 minutes is the value we chose here